torch/_functorch/partitioners.py

```diff
@@ -144,7 +144,10 @@ def __repr__(self):
 
 
 def _extract_graph_with_inputs_outputs(
-    joint_graph: fx.Graph, inputs: List[fx.Node], outputs: List[fx.Node]
+    joint_graph: fx.Graph,
+    inputs: List[fx.Node],
+    outputs: List[fx.Node],
+    subgraph: Optional[str] = None,
```

Comment:
cc @Chillee for the partitioner changes (the partitioner now needs to know if a given `copy_` node should be partitioned into the fw or bw graph, if it is a "backward mutation")